src/main/resources/documents_composition_templates/NotificationAARForPEC.html

```diff
@@ -205,9 +211,7 @@
                                             <tr>
                                                 <td align="left" style="font-size: 0px;word-break: break-word;">
                                                     <div style="font-size: 13px;line-height: 1;text-align: left;">
-                                                        <mj-raw>
-                                                            <img src="data:image/png;base64,${logoBase64}" alt="Logo SEND" class="logosend" width="116" align="left" style="display: block; width: 116px"/>
-                                                        </mj-raw>
+                                                            <img src="data:image/png;base64, ${logoBase64}" alt="Logo SEND" class="logosend" width="116" align="left" style="display: block; width: 116px"/>
```

Comment:
@leleOFA modificare il testo alternativo in "Logo SEND - Servizio Notifiche Digitali"

Reply:
done